README.md

```diff
@@ -19,7 +19,7 @@ plugins:
 ## Configuration
 ```yaml
 custom:
-  apikey: <api key name>
+  apikey: [<api key name>]
```

Comment:
shouldn't this be `apiKeys` to match with you code?

Reply:
> shouldn't this be `apiKeys` to match with you code?

My bad, fixed it.

Reply:
K should be in caps

Reply:
Done.